.gitignore

```diff
@@ -0,0 +1,160 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
```

Comment:
Please ignore just pycache since is the only useless file for this project until now